swagger/v1/openapi.json

```diff
@@ -14,12 +14,6 @@
         ],
         "description": "Lists all benchmarks requested",
         "operationId": "ListBenchmarks",
-        "consumes": [
-          "application/vnd.api+json"
-        ],
-        "produces": [
-          "application/vnd.api+json"
-        ],
```

Comment:
This happens automatically for us when we update to openapi 3.0.3 - consumes/produces isn't valid in version 3.